src/models/SearchResultsBib.ts

```diff
@@ -43,6 +43,12 @@ export default class SearchResultsBib extends Bib {
       : null
   }
 
+  numItems() {
+    return this.hasPhysicalItems
```

Comment:
Why not `this.hasPhysicalItems ?? this.numElectornicResources`?

Reply:
just moved this logic without changing it, but can clean it up if you'd like.

do you mean this.numPhysicalItems || this.numElectronicResources ?

Reply:
Hmm no. I just noticed that it's `has` and `num` + `PhysicalItems` and that made it hard to read. So ignore the comment.